Adding section to troubleshooting page.md (#6325)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md

## cloud-sync

### The projects left-most environment has changed

> The mechanism to determine changes since last deployment is not able to do so when the left-most environment has changed. This happens when you either add or remove the development environment.

Reviewer: ```suggestion
The mechanism to determine changes since the last deployment is not able to do so when the left-most environment has changed. This happens when you either add or remove the development environment.
```

The [get diff endpoint](./UmbracoCloudApi.md#get-deployment-diff) responds with status 409 and the following json payload:

```json
{
  "type":"LeftMostEnvironmentChanged",
  "title":"The projects left-most environment has changed",
  "status":409,
  "detail":"Unable to calculate changes based on a different left-most environment",
  "traceId":"..."
}
```

You will need to *manually* make sure that all latest changes on your left-most environment in cloud is also present in your local copy. 

Once this is done you can run a new deployment, where you skip the cloud-sync step.
- [Skip cloud-sync in GitHub](#skip-cloud-sync-in-github)
- [Skip cloud-sync in Azure DevOps](#skip-cloud-sync-in-azure-devops)

> If you experience problems with your development environment not properly booting up after deployment, [please read this guide](#unable-to-determine-environment-by-its-environment-id).

Reviewer: ```suggestion
If you experience problems with your development environment not properly booting up after deployment, [read the Unable to determine environment by its {environment-id} guide](#unable-to-determine-environment-by-its-environment-id).
```

4. Remove the `updating` file.

Once the marker file is removed, run your pipeline again.

## Environment errors after deployment

### Unable to determine environment by its {environment-id}

> The issue happens when you have used the CI/CD feature of Umbraco Cloud to deploy changes to your live environment, and later added a development environment to your project. Your development environment will fail to boot up and will show the following error message:

Reviewer: ```suggestion
This happens when you use the CI/CD feature of Umbraco Cloud to deploy changes to your live environment, and later add a development environment. Your development environment will fail to boot up and will show the following error message:
```
